chronicle/release/releasers/github/summarizer.go

```diff
@@ -234,34 +236,38 @@ func uniqueIssuesFromPRs(prs []ghPullRequest) []ghIssue {
 	return issues
 }
 
-func changesFromPRs(config Config, allMergedPRs []ghPullRequest, sinceTag, untilTag *git.Tag, includeCommits []string) []change.Change {
+func changesFromStandardPRFilters(config Config, allMergedPRs []ghPullRequest, sinceTag, untilTag *git.Tag, includeCommits []string) []change.Change {
```

Comment:
This slight refactoring was useful for a follow-on PR, which I erroneously worked on at the same time and have since moved to #44 -- adding a section of unlabeled PRs. I didn't want to back all these changes out just to redo them.